ida-customizations/README.md

```diff
@@ -288,7 +288,7 @@ These tutorials guide you through actionable steps and walk-throughs to teach yo
 
 The code and docs for each tutorial are based on a specific version of the software. Be sure to follow the tutorial instructions to download and use the right version.
 
-Use the tutorials landing page as your entry point to articles on [Interchain blog](https://blog.cosmos.network/), videos on [Interchain YouTube](https://www.youtube.com/c/CosmosProject/videos), and ways to get help and support.
+Use the tutorials landing page as your entry point to articles on the [Interchain blog](https://blog.cosmos.network/), videos on [Interchain YouTube](https://www.youtube.com/c/CosmosProject/videos), and ways to get help and support.

```

Comment:
It is still called the Cosmos blog. Should we leave it as that?

Reply:
Will do.